client/grpcutil/grpcutil.go

```diff
@@ -42,11 +43,11 @@ const ForwardMetadataKey = "pd-forwarded-host"
 // ctx will be noop. Users should call ClientConn.Close to terminate all the
 // pending operations after this function returns.
 func GetClientConn(ctx context.Context, addr string, tlsCfg *tls.Config, do ...grpc.DialOption) (*grpc.ClientConn, error) {
-	opt := grpc.WithInsecure() //nolint
+	creds := insecure.NewCredentials()
```

Comment:
Should we replace all `WithInsecure` with `insecure.NewCredentials()`?

Reply:
no, some grpc version is not same, the etcd can not upgrade to 1.43.